src/autoscaler/db/sqldb/appmetric_sqldb.go

```diff
@@ -71,31 +76,64 @@ func (adb *AppMetricSQLDB) SaveAppMetricsInBulk(appMetrics []*models.AppMetric)
 		adb.logger.Error("failed-to-start-transaction", err)
 		return err
 	}
-	stmt, err := txn.Prepare(CopyIn("app_metric", "app_id", "metric_type", "unit", "timestamp", "value"))
-	if err != nil {
-		adb.logger.Error("failed-to-prepare-statement", err)
-		txn.Rollback()
-		return err
-	}
-	for _, appMetric := range appMetrics {
-		_, err := stmt.Exec(appMetric.AppId, appMetric.MetricType, appMetric.Unit, appMetric.Timestamp, appMetric.Value)
+
```

Comment:
When appMetrics is empty, return nil directly, or there will be an error for mysql.